### PR TITLE
Only subscribe to FB events after a webhook has been connected

### PR DESCRIPTION
**Pull request by nicpottier**
Also allow user to configure welcome message from channel configuration page

### CODE REVIEW

**Comment by nicpottier** [comment]
plz review

File: temba/channels/tasks.py [hunk 1]
```diff
@@ -102,3 +104,18 @@ def squash_channelcounts():
...
+
+@task(track_started=True, name="fb_channel_subscribe")
+def fb_channel_subscribe(channel_id):
+    channel = Channel.objects.filter(id=channel_id, is_active=True).first()
+
+    if channel:
+        page_access_token = channel.config_json()[AUTH_TOKEN]
+
+        # subscribe to messaging events for this channel
+        response = requests.post('https://graph.facebook.com/v2.5/me/subscribed_apps',
```
Comment (norkans7) [inline temba/channels/tasks.py]: why not v2.6?

File: temba/channels/views.py [hunk 2]
```diff
@@ -1736,6 +1737,50 @@ def form_valid(self, form):
 
             return HttpResponseRedirect(reverse('channels.channel_configuration', args=[channel.id]))
 
+    class FacebookWelcome(ModalMixin, OrgPermsMixin, SmartUpdateView):
+        class WelcomeForm(forms.ModelForm):
+            message = forms.CharField(max_length=160, widget=forms.Textarea, label=_("Welcome Message"), required=False,
+                                      help_text="This message will appear when a user first interacts with your page.")
```
Comment (norkans7) [inline temba/channels/views.py]: help_text localization

**Comment by norkans7** [comment]
Looks good

**Pull request merged**